oss-ui/Makefile

```diff
@@ -0,0 +1,47 @@
+include ../metadata.mk
+
+PACKAGE_NAME = github.com/projectcalico/calico/oss-ui
```

Comment:
we're getting a lot of tendrils latching on to this name now... gotta be careful it doesn't stick!